packages/itinerary-body/src/index.js

```diff
@@ -49,14 +49,17 @@ const ItineraryBody = ({
         LegIcon={LegIcon}
         legIndex={i}
         place={leg.from}
+        PlaceName={PlaceName}
```

Comment:
These are capitalized so that they can be rendered as a higher-order components. I agree the style is confusing, but I'm open to recommendations on how to improve this.

Reply:
That's what I thought you were going to say... and it does matter if we are writing things as `<LegIcon />`; however if we are just writing `{LegIcon}`, then {legIcon} should work as well and you don't need to capitalize. (We write {children} all the time to render child tags.)

Reply:
(Hope I didn't act to quick to merge this PR.  Acted on the approved, but then noticed this conversation being so recent.)